backend/src/main/resources/schema.graphqls

```diff
@@ -174,7 +175,8 @@ type User {
   roleDescription: String!
   permissions: [UserPermission!]!
   organization: Organization
-  roles: [OrganizationRole!]
+  roles: [ApiOrganizationRole!]
+  facilities: [Facility!]
```

Comment:
Any chance we could also expose a `role` prop and an `accessAllFacilities` prop that matches what's expected in mutations? Without these elements being provided when querying for users, the frontend essentially will still have to understand how the backend role model translates to the enum role and the all facilities boolean

Reply:
I would definitely advocate for adding a singleton `role` so we can migrate to it while maintaining the single-element list `roles` as long as we need for backward compatibility.

On the other side, the `permissions` list contains the `ACCESS_ALL_FACILITIES` element—is that insufficient?

Reply:
happy to add an additional `role` field. but +1 to what Ben said about the `ACCESS_ALL_FACILITIES` element in `permissions`, i would expect that to be sufficient?

Reply:
Yes, that'll work, it'll just require some massaging to get the frontend to play nice. I was hoping to essentially be able to pass back a subset of the `User` fields when updating a user:

load user -> change field(s) -> send user back

but when we have to send the user back in a different shape, there's an additional step

load user -> convert to expected return shape -> change field(s) -> send user back

Reply:
i understand the desire for that but since, by design, we want to keep the update methods for the different parts of a user separate (eg contact info vs roles/facilities), i'm gonna keep the contract as-is, with the addition of a `role` field. sorry for the extra step!

Reply:
No problem, thanks for considering it and thanks for exposing the `role`!